server/src/main/java/keywhiz/commands/BackfillOwnershipCommand.java

```diff
@@ -33,17 +33,19 @@ public BackfillOwnershipCommand() {
   }
 
   @Override public void configure(Subparser subparser) {
+    super.configure(subparser);
+
     subparser.addArgument("--batchSize")
```

Comment:
There are multiple base classes for DropWizard commands, and you extend from them depending on the features you want. In our case we need to access the app's configuration, so we extend from the configuration-aware base class. That class operates by overriding the configure() method to add in a command line argument which is the path to the configuration file. That's great, but, if you also need to override the configure() method to add in your own command line arguments you need to remember to call super() in order to preserve the configuration file behavior.